src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralPerformanceReportRepositoryImpl.kt

```diff
@@ -127,9 +127,8 @@ class ReferralPerformanceReportRepositoryImpl : ReferralPerformanceReportReposit
   }
 
   fun timestampToOffset(timestamp: Timestamp?): OffsetDateTime? {
-    val currentTimeMillis = System.currentTimeMillis()
-    val resolved = timestamp ?: Timestamp(currentTimeMillis)
-    return OffsetDateTime.ofInstant(resolved.toInstant(), ZoneId.of("UTC"))
+    val resolved = timestamp
```

Comment:
I believe you don't need resolved variable. You can directly do `OffsetDateTime.ofInstant(timestamp?.toInstant(), ZoneId.of("UTC"))`.